templates/watcher/config/10-watcher-wsgi-main.conf

```diff
@@ -1,7 +1,7 @@
 {{ if (index . "VHosts") }}
 {{ range $endpt, $vhost := .VHosts }}
 # {{ $endpt }} vhost {{ $vhost.ServerName }} configuration
-<VirtualHost *:9311>
+<VirtualHost *:9322>
```

Comment:
Could you pass `WatchPublicPort` as a template variable here and inject it instead of hardcoding?

Reply:
sounds good, let me try that